collectors/python.d.plugin/changefinder/changefinder.chart.py

```diff
@@ -0,0 +1,175 @@
+# -*- coding: utf-8 -*-
+# Description: changefinder netdata python.d module
+# Author: andrewm4894
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from json import loads
+import re
+
+from bases.FrameworkServices.UrlService import UrlService
+
+import numpy as np
+import changefinder
+from scipy.stats import percentileofscore
+
+update_every = 1
```

Comment:
That collector does quite heavy query - `/api/v1/allmetrics?format=json` - are you sure 1 is ok as default?

Reply:
bumped it to 5 from 1.

Reply:
1 would be best in terms of performance of the algo (in sense of detecting changes) but 5 should be fine. Better to have a safer default and people can reduce it to 1 then if they want.